lightning/src/routing/scoring.rs

```diff
@@ -756,6 +793,50 @@ impl<G: Deref<Target = NetworkGraph<L>>, L: Deref, T: Time> ProbabilisticScorerU
 		None
 	}
 
+	/// Query the historical estimated minimum and maximum liquidity available for sending a
+	/// payment over the channel with `scid` towards the given `target` node.
+	///
+	/// Returns two sets of 8 buckets. The first set describes the octiles in which we've seen our
+	/// lower-bound liquidity estimates, the second set describes the octiles in which we've seen
+	/// our upper-bound liquidity estimates. Each bucket describes the relative frequency at which
+	/// we've seen a liquidity bound in the octile relative to the channel's total capacity, on an
+	/// arbitrary scale. Because the values are slowly decayed, more recent data points count for
+	/// more than older datapoints.
+	///
+	/// When scoring, the estimated probability that an upper-/lower-bound lies in a given octile
+	/// relative to the channel's total capacity is calculated by dividing that bucket's value with
+	/// the total of all buckets for the given bound.
+	///
+	/// For example, a value of `[0, 0, 0, 0, 0, 0, 32]` indicates that we belive the probability
+	/// of a bound being in the top octile to be 100%, and have never (recently) seen it in any
+	/// other octiles. A value of `[31, 0, 0, 0, 0, 0, 0, 32]` indicates we've seen the bound being
+	/// both in the top and bottom octile, and roughly with similar (recent) frequency.
```

Comment:
This part was super helpful for understanding how the historical buckets work/what they represent. I initially started reading the documentation around the `HistoricalBucketRangeTracker` and didn't really get it until I read this example. That being said, I think without having seen that implementation, it's not super clear why these numbers are 32/31. I guess it doesn't really matter for calculating the probabilities/interpreting the buckets relative to each other, and in the previous paragraphs it mentions they're on an arbitrary scale, but that was just something that stood out to me. Otherwise I thought this made sense.